### ISSUE TITLE
Having a mixed images/videos gallery displays the application instead of video

### ISSUE THREAD

**Issue by gherci** [MurhafSousli/ngx-gallery#398]
#### What is the expected behaviour?
 To have a mix of images, videos, youtube videos and when you scroll right and left to display the current video/image

#### What is the current behaviour?
When i scroll to the video and then i go back left to a picture, and then i again come back to the video instead of the video, it displays the application

#### What are the steps to reproduce?

i have the gallery implemented like this in template:
```<gallery [items]="images" gallerize></gallery>```
and in the component: 
```
this.images = [
      new ImageItem({
        src: 'https://www.gettyimages.com/gi-resources/images/500px/983794168.jpg',
        thumb: 'https://www.gettyimages.com/gi-resources/images/500px/983794168.jpg'
      }),
      new ImageItem({
        src: 'https://i.pinimg.com/originals/74/64/49/7464497f78fe88669948fe177c6916fe.jpg',
        thumb: 'https://i.pinimg.com/originals/74/64/49/7464497f78fe88669948fe177c6916fe.jpg'
      }),
      new YoutubeItem({
        type: 'video',
        src: '9dblgPmgwU8',
      }),
    ];

```

And it is functioning but with this bug. 
I have 3 pictures and 1 youtube video, when i scroll to the video and then i go back left to a picture, and then i again come back to the video instead of the video, it displays the application as you can see in the recorded.

Have this ever happened to you? What could trigger this behaviour?

![image](https://user-images.githubusercontent.com/10848502/127039172-3b6fc4c7-e5e3-4079-b439-9fe4853adaf9.png)
![image](https://user-images.githubusercontent.com/10848502/127039369-ae98beff-3e1f-499c-9885-4e47d4078711.png)
![youtubeVideoBug](https://user-images.githubusercontent.com/10848502/127123687-5d81cfcc-8490-417c-939f-432c9c22aaad.gif)

#### Which versions are you using for the following packages?

Angular:  12.1.2
Angular CDK:  12.1.3
Angular CLI:  12.1.3
Typescript: 4.3.5
Gallery: 5.0.0

**Comment by MurhafSousli** [comment]
it is working fine in the demo, please add a reproduction here https://stackblitz.com/edit/ngx-gallery

**Comment by NexGenUA** [comment]
> it is working fine in the demo, please add a reproduction here https://stackblitz.com/edit/ngx-gallery

no, it isn't
I added two videos to the demo and got the same issue
 https://stackblitz.com/edit/ngx-gallery-mcyvwt

**Comment by MurhafSousli** [comment]
@NexGenUA Thanks for the PR, there is still a bug if the slider size is smaller than the viewport width, I added a comment on your merged PR that shows the bug

**Comment by MurhafSousli** [comment]
Fixed in v5.1.1
